X/XLA_Tools/build_tarballs.jl

```diff
@@ -0,0 +1,101 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "XLA_Tools"
+version = v"2.2.1"  # latest version that doesn't require Bazel 3.x
+
+# Collection of sources required to build tar
+sources = [
+    ArchiveSource("https://github.com/tensorflow/tensorflow/archive/v$(version).tar.gz",
+                  "e6a28e64236d729e598dbeaa02152219e67d0ac94d6ed22438606026a02e0f88"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd tensorflow-*
+install_license LICENSE
+
+apk update && apk add nss
+apk add bazel --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
+
+# our Bazel is v2.2, so sufficient for TF 2.2, but it doesn't realize that.
+rm .bazelversion
+export TF_IGNORE_MAX_BAZEL_VERSION=1
+
+# the build requires numpy (really), which is only available for python3
+apk del python2 # have bazel pick up python3 instead?
+ln -s /usr/bin/python3 /usr/bin/python
+apk add py3-numpy py3-numpy-dev
+
+export TF_ENABLE_XLA=1
+export TF_NEED_JEMALLOC=0
+yes "" | ./configure
```

Comment:
How awful is that you need to this? :confused:

Reply:
Yeah it's a mess. They use a custom configure script, https://github.com/tensorflow/tensorflow/blob/v2.2.0/configure.py, but apparently it can accept env vars, so let me try that.